### PR TITLE
Add dims to test-infra-admins + test-infra-maintainers

### PR DESCRIPTION
**Pull request by dims**
Most of the folks on these 2 teams are no longer active. Putting my hands up to help with the repo. (Triggered right now by the need for adding `/lgtm`'s to some PRs that hit a github webhook hiccup!)

### CODE REVIEW

File: config/kubernetes/sig-testing/teams.yaml [hunk 1]
```diff
@@ -66,6 +66,7 @@ teams:
     - chaodaiG
     - chases2
     - cjwagner
+    - dims
```
Comment (BenTheElder) [inline config/kubernetes/sig-testing/teams.yaml:69]: we can do this in a follow up but let's review this team, we've reviewed these in OWNERS before but clearly need to go back over them here

Comment (BenTheElder) [reply config/kubernetes/sig-testing/teams.yaml:69]: I would also suggest, perhaps in another PR, that we should add @aojea and @pohly, our other SIG TLs

Comment (dims) [reply config/kubernetes/sig-testing/teams.yaml:69]: Sounds good. let me close this for now

**Pull request closed without merging**